sdk/tables/azure-data-tables/azure/data/tables/_error.py

```diff
@@ -180,23 +187,16 @@ def _decode_error(response, error_message=None, error_type=None, **kwargs):  # p
     return error
 
 
-def _reraise_error(decoded_error):
-    _, _, exc_traceback = sys.exc_info()
-    try:
-        raise decoded_error.with_traceback(exc_traceback)
-    except AttributeError as exc:
-        decoded_error.__traceback__ = exc_traceback
-        raise decoded_error from exc
-
-
-def _process_table_error(storage_error, table_name=None):
+def _process_table_error(storage_error, table_name=None, validate_key_values=False, partition_key=None, row_key=None):
     try:
         decoded_error = _decode_error(storage_error.response, storage_error.message)
     except AttributeError as exc:
         raise storage_error from exc
+    if validate_key_values:
```

Comment:
Let's keep this separate for now - we can call this function just from the one method (`create_entity`) that uses it.